WTR Sample.tex

```diff
@@ -1,22 +1,32 @@
 \documentclass[12pt]{article}
 \usepackage[utf8]{inputenc}
 
-%THIS IS WHERE ALL THE SYDE STUFF IS
+%THIS IS WHERE ALL THE SYDE MAGIK COMES FROM
 \usepackage{sydestyle}
 
+% support for colors
+\usepackage{color, soul}
+
+
 % Other imports go here
 \usepackage{graphicx}
 \graphicspath{{figures/}}
-\usepackage{notoccite}
+\urlstyle{rm}
+
+\def\reportTitle{Design of a Latex WTR Following the SYDE Guide}
+\def\reportDueDate{January 12, 1867}
+\def\studentName{Erik Derohanian}
+\def\studentNumber{12345678}
+\def\company{Gandalf Technologies}
 
 % Fill in this blob
 
-\title{Design of a Latex WTR Following the SYDE Guide}
-\author{Erik Derohanian}
+\title{\reportTitle}
+\author{\studentName}
 \studentnumber{12345678}
 \lastsemester{3A}
-\date{January 12, 1978}
-\employer{Gandalf Technologies}
+\date{\reportDueDate}
+\employer{\company}
 \employeraddress{130 Colonnade Road \\ Nepean, ON K2E 1B6}
```

Comment:
Since you're defining variables for other things, it would be nice to go further and also create variables for the employer address, your term (eg. 3B), the edition of the report (ie. first, second, final), and your address.